packages/studio/frontend/src/views/FlowEditor/diagram/nodes/Components/NodeContentItem.tsx

```diff
@@ -0,0 +1,20 @@
+import cx from 'classnames'
+import React, { FC } from 'react'
+
+import * as style from './style.module.scss'
+
+interface Props {
+  onEdit: () => void
```

Comment:
Also, this typing is kind of wrong since we forward the args sent by `onClick` to `onEdit`. If possible, can you please make those changes?